<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="installerTabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="createScriptTab">
       <attribute name="title">
        <string>Створення сценаріїв</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1041</width>
          <height>571</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="createScriptGridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="componentVerticalLayout">
           <item>
            <widget class="QLabel" name="installerComponentLabel">
             <property name="text">
              <string>Компоненти інсталятора</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="componentButtonsHorizontalLayout">
             <item>
              <widget class="QPushButton" name="openInstallerButton">
               <property name="text">
                <string>Open</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="refreshListButton">
               <property name="text">
                <string>Оновити список</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="componentButtonsHorizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTreeWidget" name="installerComponentTreeWidget">
             <column>
              <property name="text">
               <string notr="true">Компоненти інсталятора</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="proprtyComponentLabel">
             <property name="text">
              <string>Властивості компонента</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="proprtyComponentTableWidget">
             <column>
              <property name="text">
               <string>Назва</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Значення</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="ScriptsVerticalLayout">
           <item>
            <widget class="QLabel" name="scriptLabel">
             <property name="text">
              <string>Скріпт</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="hostManagerTab">
       <attribute name="title">
        <string>Менеджер локальних машин</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1031</width>
          <height>571</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="hostManagerTabGridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="hostListVerticalLayout">
           <item>
            <widget class="QLabel" name="localMachineListLabel">
             <property name="text">
              <string>Список локальних машин</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hostListButtonsHorizontalLayout">
             <item>
              <widget class="QPushButton" name="fullUpdateHostListButton">
               <property name="text">
                <string>Повне оновлення</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="updateHostListButton">
               <property name="text">
                <string>Оновлення</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="startInstalationOnHostsButton">
               <property name="text">
                <string>Запуск</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="hostListButtonsHorizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="hostListTableWidget">
             <column>
              <property name="text">
               <string>Назва хоста</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>IP</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Статус</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Для запуска</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="scriptsListVerticalLayout">
           <item>
            <widget class="QLabel" name="ScriptListLabel">
             <property name="text">
              <string>Список сценаріїв</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="scriptsListButtonsHorizontalLayout">
             <item>
              <widget class="QPushButton" name="addNewScriptButton">
               <property name="text">
                <string>Додати</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeScriptButton">
               <property name="text">
                <string>Видалити</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="editScriptButton">
               <property name="text">
                <string>Редагувати</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="updateScriptsListPushButton">
               <property name="text">
                <string>Оновити список</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="scriptsListButtonsHorizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="scriptListTableWidget">
             <column>
              <property name="text">
               <string>Назва сценарія</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Для запуска</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="installerManagerTab">
       <attribute name="title">
        <string>Менеджер інсталяторів</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1031</width>
          <height>561</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="installerManagerVerticalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="installerButtonHorizontalLayout">
           <item>
            <widget class="QPushButton" name="addInstallerPushButton">
             <property name="text">
              <string>Додати</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteInstallerPushButton">
             <property name="text">
              <string>Видалити</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editInstallerPushButton">
             <property name="text">
              <string>Редагувати</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveChangesPushButton">
             <property name="text">
              <string>Зберегти зміни</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="updateInstallerListPushButton">
             <property name="text">
              <string>Оновити список</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="installerButtonHorizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="installerLabel">
           <property name="text">
            <string>Інсталятори</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="installerTableWidget">
           <column>
            <property name="text">
             <string>Назва</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Шлях</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Аргументи</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <tabstops>
  <tabstop>openInstallerButton</tabstop>
  <tabstop>refreshListButton</tabstop>
  <tabstop>installerComponentTreeWidget</tabstop>
  <tabstop>proprtyComponentTableWidget</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>fullUpdateHostListButton</tabstop>
  <tabstop>updateHostListButton</tabstop>
  <tabstop>startInstalationOnHostsButton</tabstop>
  <tabstop>hostListTableWidget</tabstop>
  <tabstop>addNewScriptButton</tabstop>
  <tabstop>removeScriptButton</tabstop>
  <tabstop>editScriptButton</tabstop>
  <tabstop>scriptListTableWidget</tabstop>
  <tabstop>installerTabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="view.qrc"/>
 </resources>
 <connections/>
</ui>
